importer/tag/src/main/java/org/itsallcode/openfasttrace/importer/tag/LongTagImportingLineConsumer.java

```diff
@@ -55,23 +58,11 @@ public void processMatch(final Matcher matcher, final int lineNumber, final int
     {
         this.listener.beginSpecificationItem();
         this.listener.setLocation(this.file.getPath(), lineNumber);
-        final SpecificationItemId coveredId = SpecificationItemId.parseId(matcher.group(3));
-        final List<String> neededArtifactTypes = parseCommaSeparatedList(matcher.group(7));
-        final String artifactType = matcher.group(1);
-        final String revision = matcher.group(2);
-        final SpecificationItemId generatedId = generateItemId(lineNumber, lineMatchCount, coveredId, artifactType,
-                revision, neededArtifactTypes);
-
-        if (neededArtifactTypes.isEmpty())
-        {
-            LOG.finest(() -> "File " + this.file + ":" + lineNumber + ": found '" + generatedId
-                    + "' covering id '" + coveredId);
-        }
-        else
-        {
-            LOG.finest(() -> "File " + this.file + ":" + lineNumber + ": found '" + generatedId
-                    + "' covering id '" + coveredId + "', needs artifact types " + neededArtifactTypes);
-        }
+        final SpecificationItemId coveredId = SpecificationItemId.parseId(matcher.group(5));
+        final List<String> neededArtifactTypes = parseCommaSeparatedList(matcher.group(9));
+        final SpecificationItemId generatedId = createItemId(matcher, lineNumber, lineMatchCount, coveredId,
```

Comment:
Passing the matcher seems unclean to me. Better to properly assign all matching groups to a speaking name and then passing that.